src/dissolve/mdns/transport/ipv6.go

```diff
@@ -75,6 +76,13 @@ func (t *IPv6Transport) Read() (*InboundPacket, error) {
 		return nil, err
 	}
 
+	if cm == nil {
+		putBuffer(buf)
+		err := fmt.Errorf("empty control message from %s", t.Group())
```

Comment:
```suggestion
		err := fmt.Errorf("empty control message from %s", src)
```

Reply:
`t.Group()` is the "multicast group" that's being listened to, `src` is the origin of the packet.